docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -206,16 +180,15 @@ storage intact and b) you can build software before you deploy your cluster.
 
 1. Deploy the `cluster` group
 
+   Next, you can apply the `cluster` set of proposed changes on the ghpc deploy
```

Comment:
I liked the way you worded this above better where you explicitly call out "ghpc will prompt you to do x then you do y".